includes/options/class-amp-options-manager.php

```diff
@@ -562,6 +573,20 @@ public static function persistent_object_caching_notice() {
 		}
 	}
 
+	/**
+	 * Render the Stories deprecation admin notice.
+	 */
+	public static function render_stories_deprecation_notice() {
+		if ( 'toplevel_page_' . self::OPTION_NAME !== get_current_screen()->id ) {
+			return;
+		}
+
+		printf(
+			'<div class="notice notice-error"><p>%s</p></div>',
+			esc_html__( 'AMP Stories has been deprecated and will no longer be supported. A separate plugin will soon be available for testing.', 'amp' )
```

Comment:
```suggestion
			esc_html__( 'The Stories experience in the AMP plugin has been deprecated and will no longer be supported. A separate plugin for Stories be available soon for testing. Please backup or export your existing Stories as they will not be available in the next version of the AMP plugin.', 'amp' )
```

Reply:
Should we point the user to instructions on how to do the back up? (e.g. using the exporting capability)